TDDPractice/ComplicatedPage.cs

```diff
@@ -25,5 +27,22 @@ internal AmazonSearchPage SearchUsingAmazon(string searchTerm)
             Driver.SwitchTo().Window(Driver.WindowHandles[1]);
             return new AmazonSearchPage(Driver);
         }
+
+        internal void SearchArticles(string searchTerm)
```

Comment:
I see you added some new tests here. I guess it's using our Instant Search feature?

Reply:
@nadvolod yes, this is a replacement for the Amazon search, which was removed. I will also add a text lecture for this, once the code is available to the students